doc/backend/README.md

```diff
@@ -10,6 +10,9 @@
 
 ## Additional Documentation
 
+Instead of setting up psql server on your local machine, you could use a 'postgis' Docker container. Details about the setup can be found [here](/setups/postgis/)
```

Comment:
```suggestion
Instead of setting up psql server on your local machine, you could use a 'postgis' Docker container.
Details about the setup can be found [here](/setups/postgis/)
```